frontend/src/app/modules/auth/login-page/login-page.component.ts

```diff
@@ -9,52 +9,54 @@ import { ExternalAuthService } from 'src/app/services/external-auth.service';
 @Component({
   selector: 'app-login-page',
   templateUrl: './login-page.component.html',
-  styleUrls: ['./login-page.component.scss']
+  styleUrls: ['./login-page.component.scss'],
 })
 export class LoginPageComponent implements OnInit {
-
-  @Input() userLoginDto : UserLoginDto = {} as UserLoginDto;
-  public loginForm : FormGroup = {} as FormGroup;
+  @Input() userLoginDto: UserLoginDto = {} as UserLoginDto;
+  public loginForm: FormGroup = {} as FormGroup;
 
   public hidePass = true;
   public hideConfirmPass = true;
-  public currentUser:UserDto = {} as UserDto;
+  public currentUser: UserDto = {} as UserDto;
 
   constructor(
-    private router : Router,
-    private formBuilder : FormBuilder,
-    private loginService : LoginService,
+    private router: Router,
+    private formBuilder: FormBuilder,
+    private loginService: LoginService,
     private externalAuthService: ExternalAuthService
-  ) { }
+  ) {}
 
   ngOnInit() {
     this.validateForm();
   }
 
   private validateForm() {
     this.loginForm = this.formBuilder.group({
-      email: [, {
-        validators: [
-          Validators.required,
-          Validators.email
-        ],
-        updateOn: 'change',
-      }],
-      password: [, {
-        validators: [
-          Validators.required,
-          Validators.minLength(8),
-          Validators.pattern('^(?=.*[0-9])(?=.*[a-zA-Z])([a-zA-Z0-9]+)$'),
-        ],
-        updateOn: 'change'
-      }],
+      email: [
```

Comment:
Check again the formatting of this file. I think you can get rid of those commas. And probably the wrapping object also